data/json/vehicleparts/alternator.json

```diff
@@ -0,0 +1,84 @@
+[ 
+  {
+    "abstract": "vehicle_alternator",
+    "type": "vehicle_part",
+    "fuel_type": "battery",
+    "symbol": "*",
+    "color": "yellow",
+    "broken_color": "red",
+    "flags": [ "ALTERNATOR" ]
+  },
+  {
+    "id": "alternator_motorbike",
+    "copy-from": "vehicle_alternator",
+    "type": "vehicle_part",
+    "name": "motorbike alternator",
+    "item": "alternator_motorbike",
+    "difficulty": 2,
+    "damage_modifier": 80,
+    "durability": 250,
+    "power": -2,
+    "epower": 360, "//": "30A @12VDC ~ 1HP drain @ 48% effficiency",
+    "folded_volume": 5,
+    "breaks_into": [
+        { "item": "steel_lump", "count": [ 2, 5 ] },
+        { "item": "steel_chunk", "count": [ 2, 5 ] },
+        { "item": "scrap", "count": [ 2, 5 ] }
+    ],
+    "extend": { "flags": [ "FOLDABLE", "TOOL_SCREWDRIVER" ] }
+  },
+  {
+    "id": "alternator_car",
+    "copy-from": "vehicle_alternator",
+    "type": "vehicle_part",
+    "name": "car alternator",
+    "item": "alternator_car",
+    "difficulty": 2,
+    "damage_modifier": 80,
+    "durability": 275,
+    "power": -4,
+    "epower": 780, "//": "65A @12VDC ~ 2HP drain @ 52% effficiency",
+    "breaks_into": [
+        { "item": "steel_lump", "count": [ 3, 6 ] },
+        { "item": "steel_chunk", "count": [ 3, 6 ] },
+        { "item": "scrap", "count": [ 3, 6 ] }
+    ],
+    "extend": { "flags": [ "TOOL_WRENCH" ] }
+  },
+  {
+    "id": "alternator_truck",
+    "copy-from": "vehicle_alternator",
+    "type": "vehicle_part",
+    "name": "truck alternator",
+    "item": "alternator_truck",
+    "difficulty": 2,
+    "damage_modifier": 80,
+    "durability": 325,
+    "power": -6,
+    "epower": 1320, "//": "110A @12VDC ~ 3HP drain @ 60% effficiency",
+    "breaks_into": [
+        { "item": "steel_lump", "count": [ 4, 7 ] },
+        { "item": "steel_chunk", "count": [ 4, 7 ] },
+        { "item": "scrap", "count": [ 4, 7 ] }
+    ],
+    "extend": { "flags": [ "TOOL_WRENCH" ] }
+  },
+  {
+    "id": "generator_7500w",
```

Comment:
It would be good to change "7500W" part of names by something more abstract/generic. It will allow us to rebalance this item easier without thinking of id change (and avoid discussions like one we had recently about missleading id).